less/home-page.less

```diff
@@ -1 +1,101 @@
-// Home Content
\ No newline at end of file
+// Home Content
+.home {
+    .earth {
+        display: flex;
+
+        img {
+            width: 100%;
+            padding-bottom: 20px;
+        }
+    }
+
+    .lift-off {
+        .flex-wrap();
+        justify-content: space-between;
+        padding: 0 3% 50px 3%;
+
+        h2 {
+            padding-bottom: 10px;
+        }
```

Comment:
you have `padding-bottom: 10px;` for h2 multiple times. Did you consider using a variable or placing it in `global.less`?